Classes/TestableFramework.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * (c) Contributors of the Neos Project - www.neos.io
+ * Please see the LICENSE file which was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\BuildEssentials;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Configuration\ConfigurationManager;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Core\RequestHandlerInterface;
+use Neos\Flow\Persistence\PersistenceManagerInterface;
+use Neos\Utility\Files;
+
+/**
+ * Low level operations for use in testing, all to be used in static calls.
+ *
+ * @Flow\Proxy(false)
+ */
+abstract readonly class TestableFramework
```

Comment:
thanks for making things more descriptive but this is not a net plus right? We cannot reuse any of this logic for other tests or can we? But maybe we can remove sprinkles from the flow functional tests with this class?

Reply:
That is the idea? It's really just boilerplate code, I will do more changes for functional tests once this is in and I can use it in Flow.

Reply:
perfecto ;) In that case i didnt say a think ;)

Reply:
Can i help with the planned things?